### PR TITLE
BUGFIX: Setting a camera in the view makes the view blank.

### PR DESCRIPTION
**Pull request by pm-ur**
This is a partial fix for #14 . Now I can get the rendering if I set the scene data later, but the rendering is still weird. This might give some hints to fix it further.

### CODE REVIEW

**Comment by podsvirov** [comment]
Hey @pm-energid! Thanks for PR. Unfortunately, I'm on vacation now. I will review your changes when I return to my usual schedule.

**Comment by pm-ur** [comment]
Sure Konstantin, This is not a complete fix and only fixes a part of the
problem. But I hope this update helps getting closer to the full fix.

File: src/quick/osgviewer/view.cpp [hunk 1]
```diff
@@ -105,6 +105,8 @@ osg::CameraQtQml *ViewQtQuick::Index::getCamera()
 void ViewQtQuick::Index::setCamera(osg::CameraQtQml *camera)
 {
     o(this)->setCamera(camera->camera());
+    o(this)->getCamera()->setPreDrawCallback(preDraw.get());
+    o(this)->getCamera()->setPostDrawCallback(postDraw.get());
```
Comment (podsvirov) [inline src/quick/osgviewer/view.cpp:109]: Please add also
```cpp
o(this)->getCamera()->setGraphicsContext(context);
```

Comment (podsvirov) [reply src/quick/osgviewer/view.cpp:109]: Ok. It's done by me.
Please review new [camera.qml](https://github.com/podsvirov/osgqtquick/blob/develop/tests/camera.qml) test.

**Pull request merged**